homeassistant/components/eq3btsmart/entity.py

```diff
@@ -0,0 +1,18 @@
+"""Base class for all eQ-3 entities."""
+
+from eq3btsmart.thermostat import Thermostat
+
+from homeassistant.helpers.entity import Entity as HassEntity
+
+from .models import Eq3Config
+
+
+class Entity(HassEntity):
+    """Base class for all eQ-3 entities."""
+
+    def __init__(self, eq3_config: Eq3Config, thermostat: Thermostat) -> None:
+        """Initialize the eq3 entity."""
+
+        self._eq3_config = eq3_config
+        self._thermostat = thermostat
+        self._attr_name = self._eq3_config.name
```

Comment:
Since we are using has_entity_name should we remove this?

Reply:
This will be used by the other platforms that I'll add later, but I can remove it for now.